pegasus/sites.v3/code.org/views/promote_tools.haml

```diff
@@ -3,13 +3,14 @@
   -if value[:title_s] == 'Convince your school or state' && ! state.nil_or_empty? && File.exists?(sites_dir("all/files/states/#{state}.pdf"))
     -url = "/files/states/#{state}.pdf"
 
-  #block{:style=>'width:20%; float:left; padding:30px; padding-top:0px; padding-bottom:0px'}
-    %a{:href=>url, :target=>'_blank', :style=>'text-decoration:none'}
-      %img{:src=>value[:image_url_s], :style=>"width: 60%"}
-    #text{:style=>'float:left; width:75%'}
-    %br
-    %br
-    %a{:href=>url, :target=>'_blank'}
-      =value[:title_s]
-    %p{:style=>'font-size:13px; line-height:18px'}
-      =value[:description_t]
+  %div.col-18eb{:style=> 'margin:1%; float:left; height:275px;'}
```

Comment:
Cool shortcut with HAML, you don't need to start with `%div.col-18eb` you can just do `.col-18eb`.